cvat/apps/engine/task.py

```diff
@@ -262,20 +268,18 @@ def _create_thread(tid, data):
     validate_dimension = ValidateDimension()
     if extractor.__class__ == MEDIA_TYPES['zip']['extractor']:
         extractor.extract()
-        validate_dimension.set_path(os.path.split(extractor.get_zip_filename())[0])
-        validate_dimension.validate()
-        if validate_dimension.dimension == DimensionType.DIM_3D:
-            db_task.dimension = DimensionType.DIM_3D
-
-            extractor.reconcile(
-                source_files=list(validate_dimension.related_files.keys()),
-                step=db_data.get_frame_step(),
-                start=db_data.start_frame,
-                stop=data['stop_frame'],
-                dimension=DimensionType.DIM_3D,
-
-            )
-            extractor.add_files(validate_dimension.converted_files)
+    validate_dimension.set_path(upload_dir)
+    validate_dimension.validate()
```

Comment:
1) All data on the share point is validated (if a share is used without copying).
2) Accordingly, if there is data without extension, then the task is not created.

**Example data share structure:**

 - sub_images/
   - image_without_extention
   - image_2.jpeg
   - image_3.jpeg
  - **image_1.jpeg (selected)**
 - **image_2.jpeg (selected)**
```
Traceback (most recent call last):
File "/home/maya/Documents/cvat/.env/lib/python3.8/site-packages/rq/worker.py", line 936, in perform_job
rv = job.perform()
File "/home/maya/Documents/cvat/.env/lib/python3.8/site-packages/rq/job.py", line 684, in perform
self._result = self._execute()
File "/home/maya/Documents/cvat/.env/lib/python3.8/site-packages/rq/job.py", line 690, in _execute
return self.func(*self.args, **self.kwargs)
File "/usr/lib/python3.8/contextlib.py", line 75, in inner
return func(*args, **kwds)
File "/home/maya/Documents/cvat/cvat/apps/engine/task.py", line 272, in _create_thread
validate_dimension.validate()
File "/home/maya/Documents/cvat/cvat/apps/engine/media_extractors.py", line 856, in validate
self.validate_default(root, actual_path, files)
File "/home/maya/Documents/cvat/cvat/apps/engine/media_extractors.py", line 821, in validate_default
pcd_files = self.process_files(root, actual_path, files)
File "/home/maya/Documents/cvat/cvat/apps/engine/media_extractors.py", line 758, in process_files
file_name, file_extension = file.rsplit('.', maxsplit=1)
ValueError: not enough values to unpack (expected 2, got 1)
```

On develop branch such case not reproduced, because the validation is started only for `zip` media type.